pages/agent/v3/help/_env_dump.md

```diff
@@ -23,14 +23,21 @@ that hooks make to the environment.
 
 ### Example
 
-     $ buildkite-agent env dump --format json-pretty
+```shell
```

Comment:
These `shell` code blocks look a bit nicer than plain indented code blocks. I plan to make the script output them all the time.